screens/Login/Login.component.js

```diff
@@ -7,32 +7,39 @@ import MainButton from '../../components/MainButton/MainButton.component';
 import { TouchableOpacity } from 'react-native-gesture-handler';
 import TextLabel from '../../components/TextLabel/TextLabel.component';
 
+import { UserContext } from '../../context/UserContext';
 const axios = require('axios');
 
 const Login = ({navigation}) => {
 
 	const [email, setEmail] = useState('');
 	const [password, setPassword] = useState('');
-
+	const { token, setToken} = useContext(UserContext);
 	const login = async () => {
-		const url = 'https://jsonplaceholder.typicode.com/posts';
+		const url = 'http://10.0.2.2:9090/login';
```

Comment:
APIs dont work unless you have the python django backend running locally

Reply:
You need to set up https://github.com/cpp-css/backEndFall2020 and have python,django, and postgres installed
Probably shouldnt merge

Reply:
I'll review it once we have the majority of the team has the backend up and running.

Reply:
We should have an .env file that stores the localhost URL, so whenever the address or port changes we don't need to go in and manually change every one of them.